doc/usage.rst

```diff
@@ -128,15 +128,15 @@ It is available for most operating systems and can compress a large number
 of executable file formats. See the UPX_ home page for downloads, and for
 the list of supported file formats.
 
-When UPX is available, |PyInstaller| uses it to individually compress
+When UPX is available, PyInstaller uses it to individually compress
 each collected binary file (executable, shared library, or python
 extension) in order to reduce the overall size of the frozen application
 (the one-dir bundle directory, or the one-file executable). The frozen
 application's executable itself is not UPX-compressed (regardless of one-dir
 or one-file mode), as most of its size comprises the embedded archive that
 already contains individually compressed files.
 
-|PyInstaller| looks for the UPX in the standard executable path(s) (defined
+PyInstaller looks for the UPX in the standard executable path(s) (defined
```

Comment:
This section should mention (if we don't do so somewhere else...) that UPX is actually used only on Windows.

Reply:
Later PR.